src/lambda_function.py

```diff
@@ -53,6 +53,13 @@ def lambda_handler(event, context):
     print "LOG: The request is: " + str(text)
     print "LOG: The requesting user is: " + param_map['user_name']
 
+    #extract send to slack channel from args
+    sendto_data = None
+    if "sendto:" in text:
```

Comment:
Did we update the help to include the fact you can now sendto a channel?  I assume this is generic now to all commands?